packages/governance/src/contractGovernor.js

```diff
@@ -304,6 +309,8 @@ const start = async (zcf, privateArgs) => {
     });
   };
 
+  /** @type {GovernorCreatorFacet<SF>} */
+  // @ts-expect-error cast
```

Comment:
this error is a little surprising; aren't we supplying exactly what the type requires?

Reply:
It surprised me too. Apparently within the function TS will only trust the minimum of the parameter (what SF is said to extend) which is `{}`. I tried doing this without an error but timed out